modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prequisites
+. You installed the OpenShift CLI (`oc`).
+. You installed Custom Metrics Autoscaler Operator.
+. You enabled OpenShift monitoring for user-defined projects.
```

Comment:
For all the steps I mentioned we need to also reference the yanl content as well as any `oc` command that either applies the yaml or runs specific oc commands.

Reply:
@jldohmann I split up the README in https://github.com/frobware/openshift-keda-examples. The README.md now has the steps that I believe we need for the documentation. All the other "guff" I moved into custom.md and is only for debugging/experimentation so you can freely ignore it. HTH.

Reply:
Thanks @frobware, this is super helpful. I've moved some of the contents out of the prerequisite section and into the procedure proper.

The only one I didn't include in the actual procedure is the installation, which is [covered here already](https://docs.openshift.com/container-platform/4.11/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-install_nodes-pods-autoscaling-custom). 

Other portions of the procedure are also covered in those docs, but having them contextualized for the ingress controller is a good idea to show the complete workflow 🙂